git-config/tests/fixtures/config_with_worktree_extension.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eu -o pipefail
+
+git init -q
+
+git config extensions.worktreeConfig true
+git config --worktree someSection.someSetting "some value"
```

Comment:
This would better be a meaningful section name and value, to indicate it comes from the worktree config. That way the test is more obvious and readable. _I mention this for adaptation in `git-repository`_.